Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/klipper/readme/) for klipper/readme.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/klipper/-/readme/horizontal-auto.svg)

fitojb commented:
If not done already, I recommend setting up [the Weblate addon](https://docs.weblate.org/en/latest/admin/addons.html#squash-git-commits) so that it squashes commits by author, in order to reduce the amount of tiny commits that get added to the repo

weblate committed:
localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

localization: Added translation using Weblate (Turkish)

Co-authored-by: Weblate <noreply@weblate.org>
Signed-off-by: Yifei Ding <yifeiding@protonmail.com>

udopton committed:
localization: Translated using Weblate (Turkish)

Currently translated at 11.7% (4 of 34 strings)

Translation: klipper/Installation
Translate-URL: https://hosted.weblate.org/projects/klipper/installation/tr/

localization: Added translation using Weblate (Turkish)

Co-authored-by: udo pton <udopton@gmail.com>
Signed-off-by: Yifei Ding <yifeiding@protonmail.com>

fitojb committed:
localization: Translated using Weblate (Spanish)

Currently translated at 94.2% (33 of 35 strings)

Translation: klipper/Debugging
Translate-URL: https://hosted.weblate.org/projects/klipper/debugging/es/

localization: Translated using Weblate (Spanish)

Currently translated at 28.5% (10 of 35 strings)

Translation: klipper/Debugging
Translate-URL: https://hosted.weblate.org/projects/klipper/debugging/es/

Co-authored-by: Adolfo Jayme Barrientos <fitojb@ubuntu.com>
Signed-off-by: Yifei Ding <yifeiding@protonmail.com>

dingyifei commented:
Thank You, I'll be doing that. Additionally, I'll apply for librehosting after I figure out the screen shots
